src/3_PB/VerbaliInterni/VerbaleInterno_240215/changelog.csv

```diff
@@ -1 +1 @@
-0.1,00-00-0000,Descrizione sintetica senza virgole,Autore,Ruolo
\ No newline at end of file
+0.1,19-02-2024,Prima stesura,Matteo Rango,Responsabile
```

Comment:
volendo aggiungere la colonna per il verificatore come fatto altrove:

in changelog.typ
#let changelog = csv("changelog.csv")
#table(
  columns: (1fr, 2fr, 4fr, 2fr, 2fr, 2fr),
  [*Versione*], [*Data*], [*Descrizione*], [*Autore*], [*Ruolo*], [*Verificatore*],
  ..changelog.rev().flatten()
)